interlok-core/src/main/java/com/adaptris/core/services/path/XPathService.java

```diff
@@ -248,11 +248,17 @@ private String serializeNode(NodeList nodeList) throws TransformerException {
   }
   
   private String serializeNode(Node node) throws TransformerException {
-    StreamResult xmlOutput = new StreamResult(new StringWriter());
-    Transformer transformer = TransformerFactory.newInstance().newTransformer();
-    transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
-    transformer.transform(new DOMSource(node), xmlOutput);
-    return xmlOutput.getWriter().toString();
+	  DOMSource source = new DOMSource(node);
+	    StringWriter stringWriter = new StringWriter();
+		StreamResult xmlOutput = new StreamResult(stringWriter);
+	    Transformer transformer = TransformerFactory.newInstance().newTransformer();
+	    transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+	    if (source.getNode().getNodeType() != Node.ATTRIBUTE_NODE) {
+	        transformer.transform(source, xmlOutput);
+	      } else {
+	        stringWriter.write(source.getNode().getNodeValue());
+	      }
+	    return stringWriter.toString();
```

Comment:
Spaces not tabs... (when I view the while file, this sticks out like a sore thumb).

Reply:
Apologies, just updated the indentations to two spaces